WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -130,6 +130,24 @@ private extension ProductsViewController {
     }
 }
 
+// MARK: - Navigation Bar Actions
+//
+private extension ProductsViewController {
+    @IBAction func displaySearchProducts() {
+        guard let storeID = ServiceLocator.stores.sessionManager.defaultStoreID else {
+            return
+        }
+
+        // TODO-1263: analytics
+        let searchViewController = SearchViewController<ProductsTabProductTableViewCell, ProductSearchUICommand>(storeID: storeID,
```

Comment:
It is possible to remove the brackety thingy 😛 so this could read like:

```
let searchViewController = SearchViewController(storeID: storeID....
```

I think it is more readable, but that's just me 🤷‍♂

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/1315/commits/16a57763f6e47a0b74082405bce0ef90b0e2972c